src/dotty/tools/dotc/transform/GetClass.scala

```diff
@@ -0,0 +1,42 @@
+package dotty.tools.dotc
+package transform
+
+import ast.tpd
+import core.Contexts.Context
+import core.StdNames.nme
+import core.Symbols.TermSymbol
+import TreeTransforms.{MiniPhaseTransform, TransformerInfo}
+
+/**
+ * FILL ME
+ */
+class GetClass extends MiniPhaseTransform {
+  import tpd._
+
+  override def phaseName: String = "getClass"
+
+  override def transformApply(tree: Apply)(implicit ctx: Context, info: TransformerInfo): Tree = {
+    import ast.Trees._
+
+    tree match {
+      case Apply(Select(qual, nme.getClass_), Nil) =>
+        val defn = ctx.definitions
+        val claz = qual.tpe.classSymbol
```

Comment:
I thought that order in `Compiler` is enough for phases ordering. Why do we explicitly need `runsAfter`?